src/collections/src/NICollectionViewModel.h

```diff
@@ -105,26 +105,26 @@
  * The value of the kind property and indexPath are implementation-dependent
  * based on the type of UICollectionViewLayout being used.
  */
-- (UICollectionReusableView *)collectionViewModel:(id<NICollectionViewModeling>)collectionViewModel
-                                   collectionView:(UICollectionView *)collectionView
-                viewForSupplementaryElementOfKind:(NSString *)kind
-                                      atIndexPath:(NSIndexPath *)indexPath;
+- (nullable UICollectionReusableView *)collectionViewModel:(nonnull id<NICollectionViewModeling>)collectionViewModel
```

Comment:
I think the return type should be nonnull to match -collectionView:viewForSupplementaryElementOfKind:atIndexPath: https://developer.apple.com/documentation/uikit/uicollectionviewdatasource/1618037-collectionview?language=objc

Reply:
Done.